property_tests/tests/stable_b_tree_map/test/test.ts

```diff
@@ -65,8 +65,8 @@ const StableBTreeMapTestArb = fc
                 );
         }),
         {
-            minLength: 1,
-            maxLength: 50
+            minLength: 20,
+            maxLength: 100
```

Comment:
`defaultArrayConstraints` in azle/property_tests is 20-100, consider using that instead.